core/controller/src/whisk/core/entitlement/LocalEntitlement.scala

```diff
@@ -31,9 +32,11 @@ private object LocalEntitlementService {
 
 protected[core] class LocalEntitlementService(
     private val config : WhiskConfig)
-    (implicit ec: ExecutionContext)
+    (implicit actorSystem: ActorSystem)
     extends EntitlementService(config) {
 
+    private implicit val executionContext = actorSystem.dispatcher
```

Comment:
Seems like the `LocalEntitlementService` doesn't need an actor system, so rather just keep the `ExecutionContext` here?